docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+kube-controller-manager-ip-172-20-79-203.ec2.internal   1/1       Running   0          7m
+kube-dns-479524115-28zqc                                3/3       Running   0          8m
+kube-dns-479524115-7xv6b                                3/3       Running   0          6m
+kube-dns-autoscaler-1818915203-zf0gd                    1/1       Running   0          8m
+kube-proxy-ip-172-20-125-216.ec2.internal               1/1       Running   0          6m
+kube-proxy-ip-172-20-125-90.ec2.internal                1/1       Running   0          7m
+kube-proxy-ip-172-20-48-12.ec2.internal                 1/1       Running   0          3m
+kube-proxy-ip-172-20-79-203.ec2.internal                1/1       Running   0          7m
+kube-proxy-ip-172-20-92-185.ec2.internal                1/1       Running   0          7m
+kube-scheduler-ip-172-20-125-90.ec2.internal            1/1       Running   0          7m
+kube-scheduler-ip-172-20-48-12.ec2.internal             1/1       Running   0          3m
+kube-scheduler-ip-172-20-79-203.ec2.internal            1/1       Running   0          8m
+
+```
+
+
+## LAUNCH A SIMPLE REPLICATED APP ON THE CLUSTER.
+
+Before doing the tasks ahead, we created a simple "webservers" security group inside our KOPS's cluster VPC (using the AWS WEB-UI) allowing inbound port 80 and applied it to our two nodes (not the masters). Then, with the following command we proceed to create a simple replicated app in our coreos-based kops-launched cluster:
+
+```
+kubectl run apache-simple-replicated \
+--image=httpd:2.4-alpine \
+--replicas=2 \
+--port=80 \
+--hostport=80
+```
+
+Then check it:
+
+```bash
+kubectl get pods -o wide
+NAME                                        READY     STATUS    RESTARTS   AGE       IP           NODE
+apache-simple-replicated-1977341696-3hxxx   1/1       Running   0          31s       100.96.2.3   ip-172-20-92-185.ec2.internal
+apache-simple-replicated-1977341696-zv4fn   1/1       Running   0          31s       100.96.3.4   ip-172-20-125-216.ec2.internal
+```
+
+Using our public IP's (the ones from our kube nodes, again, not the masters):
+
+```bash
+curl http://54.210.119.98
+<html><body><h1>It works!</h1></body></html>
+
+curl http://34.200.247.63
+<html><body><h1>It works!</h1></body></html>
+
+``` 
+
+Now, let's delete our recently-created deployment:
+
+```bash
+kubectl delete deployment apache-simple-replicated
+```
+
+NOTE: In the AWS Gui, we also deleted our "webservers" security group (after removing it from out instance nodes).
+
+Check again:
+
+```bash
+kubectl get pods -o wide
+No resources found.
+```
+
+Finally, let's destroy our cluster:
+
+```
+kops delete cluster ${NAME} --yes
+```
+
+After a brief time, your cluster will be fully deleted on AWS and you'll see the following output:
+
+```bash
+Deleted cluster: "coreosbasedkopscluster.k8s.local"
+```
+
+NOTE: Before destroying the cluster, "really ensure" any extra security group "not created" directly by KOPS has been removed by you.
```

Comment:
Ok !. I'll take all recommendations into account and do the changes to the documentation. All sound OK to me !.